extpsdk/build.gradle

```diff
@@ -10,6 +10,9 @@ ext {
     } else {
         pypi_repo = 'testpypi'
     }
+    if (!project.rootProject.hasProperty('vantiqConnectorSdkVersion')) {
+        vantiqConnectorSdkVersion = '1.1.9'
```

Comment:
Because, to test & use everything for the exec connector, it was published a few times with earlier versions.  I don't like to leave things in a strange state on the test.pypi repo, so I just keep the versions consistent.  Otherwise, have to go & delete old things -- which seems to be frowned upon by the community.